src/ui/main.qml

```diff
@@ -6,6 +6,8 @@ import QtQuick 2.5
 import QtQuick.Controls 2.14
 import QtQuick.Layouts 1.14
 import QtQuick.Window 2.12
+import Qt5Compat.GraphicalEffects
```

Comment:
This line seems to break qt5 compile support ... but without it the windows+qt6 client will crash 😅

Reply:
Could we create a build variable just like MVPN_IOS and others for Qt5 and Qt6?

Reply:
We agreed it is fine to break Qt5.

Reply:
This is just a temporary fix of a bigger issue. Yes, the app starts, but it will crash in the following steps:
- opening the dev menu -> qtcharts2 plugin not found
- completing the auth -> graphical effects plugin not found
- on the main view -> localstorage plugin not found

Reply:
See the latest `main` commits. This is now fixed.